doc/source/examples/pdp_regression_bike.ipynb

```diff
@@ -0,0 +1,686 @@
6Ua9SaVbtOQrAyfOx/LNxL9/+r0+G/TvWq0LPJjXw8bIyZdk2Xpq0gG+c+/39wm2UDSnB4TNRPPD1DGqVC6dyqczve8aaXWw9fJrvHrp4jdwc7w669ejE6VNn2Lh+C23bt8zxcf1v6MX0P+ZgtztSxP/+XUKTplczfc5Ezpw5y+qV67E7k+miIq/jd9yNJsMeJjklES9r1jWvmSrkn1kRoVm7/P0q7oKU6HiuqnPls1OkJKZkmQwnJSTj66+VQ8VVftcNu5GPgLXA92natgLNgH/TtDUDLtQPLAXKi0gjoC2OhDgDY8wYHKUYACY/l2Nev30PC1ZtZPGazSQmpxAbF8+Ax17Hx9uLPv97CYCExCR6/+8lZnw1kjLhoXz43P8AiItPYO6ydQQHOnpBh9zUiyE39QLg2fe/I6JCmQzXK1sylAa1q+HtZaVS2VJUrVCGg8dOcnWtiNR9yoSHUrNKRdZs3UX3ts3y7V7z0/r9J/hv60EWb59IUrKN2MQkbnh/Cj5WK33f+QWAhOQU+o76hb+eu4XQNN+0DWhVh4/+XpH6umyIo3e0Uslgmlcvz/ajpzNNZpfvPMK3/67nu4f64ONlzfXx7qJl66b06HktXbp1xM/Ph8CgQN54ZwTBIcFYrVZsNhsVKpTj2LGT6Y7rf0MvnhuWfpDbR+99zUfvOcruv/z2Pfbs3p/heseOnaRixfIcO3oCq9VKUHCQR/QKQ9GrGdZk2MMk25Jz3zNchDo7k85GEZCHQUkpiSlZDqCLi0ogMNia6TZVtFXwj+Bo/H5Xh1EgjDFnRWQycB8w1tn8LvCOiFxnjDkjIo2BwUAr5zFGRH7BUUc80xiTkPHMBeuJuwbwxF0DAFi1aQfj/5ibbjYJgFa3PsGMrxxJyIVZJCwWC9/+NosBXdoCjhrj6Ng4QoMD2bn/MDsPHKFNk3oZrte5VWNmLlpF/y5tORcVw4GjJ6lUthTHT58jNKgEfr4+RMXEsm7bbu7s26WA7/7KPd6rJY/3cvRqrtpzlB/+25huNgmANiO+56/nHGVsp6LiUksn/ttygGplHL9fo+IS8fPxwsfLyrnYBNYfOMHgzo0yXG/7kdO88dsiPr+/J+FpvrXL6fHu5M3XPuDN1z4AoG37ljz82L08/MBwvhn/EX3792Dab39z8+39mfX3vNRjataqRkhICKtXrktts1gshIQEc+5cJPXq16Ze/dos+HdJhuvN/vtfbr69P6tXradv/x4sXri84G8yn2iZhHKpFFsSVjfuGV5nThTo+ZPPx+AXGpSnc2Q1V3NcZDzBofpXQhVJ7wOPXnhhjPlTRCoCS51TsEUDdxpjjqU5ZiLwDPBcoUZ6hVZt3sEnP05DRGharxYjHnR0ZqfYbAx+4T0ASgT48/aT9+BldXzo/fznP6lXsyqdWzaiXZN6LFu/lf6PvorFYmHo4IGEBgeybP1W3vv+N0Qci/IM6teN2hEVXXaf+W3i4s0s2HoAL4uF4ABfXr/FMR3Y3pORvPHbIiwi2I3h3s6NqFHWkSh/MXs19SqVplP9qnw4fQVxSSkM/3EucHEKteyO9zRvvPIeX4/9gOdefIJNG7fx8w9TUrf1v6E3f0ydkW5/b28v/pj1EwAx0TE8POSZ1JrjZ154jA3rNjN75nx+/nEKn415l+XrZhN57jwP3nux/GfVxnkEBZfAx9ubnr27cMuA+9i5I2Pdsau4+4C43JKiVvcBUC68ihnU5WlXh1EgFhzYSEStypSvmPPpVn7/70e63nlPAUZ10TpzgroVM34FmV8W/jqGHo/ffsXHbx73OYOfuT7TbQkrJuLlJfTs1OCKz68829H4/QVeKvFwrVfXGGMyn+jWs+VrmURx4ntVZ+L/fM/VYXgk/+ufpmxIXVeH4ZFOnN9+xd8b96rSK9fJ498H/3bb76m1QNLD2GzJeHkV3zKJPMtmIuGoyBRqlK1SiMEopZRSnscYk+uHO9Nk2MPYbcl4eed8ajVj7EghZcMF3Stc0KLPpxAS5t4zdaiCV99rxuV3UkqpYsx+BQ93psmwh7EbGxZLzgd52e12JJNphIqtbD6dxkbbCAzKZT22KlIKYgEOpZQqanTRDeVSxpgsB4Blvr8dpGgkw/aUFMRacPeSmGDH319nk1BKKaWyo1OrKRczSC6S29wmz+4sOT4Ba8CVrz4HZFszbAxYLEXjvVJKKaUKirvXAOdW0egyLEaMMbkbEGdMtgmgJ0mOj8PqX3BLURexv9sqD7RuWCmlsmbH5PrhzjQZ9ji57BmmcHqGC3p+YYCU+IQCTYaVAq0bVkqpy9GaYeVSxthzlwzb7RTW3GoFPZPEgfhdeJXwv/yO2dHuX6UKjO9VnV0dgsfyv75ozo1fGE6c3+7qEIodexH7t1STYQ+TlBSHv39gjvdPTk7E6l00pguLO3iC1n06XvHxdpsdyaYmuIhUkyjlMolb5l1+J5WBb/0uxE94ydVheCT/O0YSXKK6q8PwSFGxe6/42KKVCmsy7HGMsedqarWkpHisPnkcdHYZhVEiAZAcGU1AyZArPz42Dt/ArMssitgHXaWUUqpAuHsNcG5pzXARl5SUgDUXi3RcKU9YbCMxKha/UK05VkoppdRF2jNcxG2NO0mViPIFdv7C6hW2Jacg1rzNAZwYFYtfcNa95FomoZRSSl1eUesZ1mS4iLMlJeLtV7BlEoXRK3z+0CECqpTN0zkkfi11KlTMp4iUUkqp4knnGVYucyU/fLbkRLx9CzYZLgw796+kccsGeTpH/Nl4gsIDstxexP5uK6WUUgVC5xlWLpOUkoiXV+5qXgsyGV5nThRarXDSqXOUKFsyT+eIORVLSNmgTLfZbXYs+rdBOVXwjygyC2+ISCUR+UNEdonIHhH5WER8nNtaishCEdkhIutE5FsReURE1jsfSSKyyfl8lCvit9ns3DzsLR5984t07aO+nUyr259KfX305Bnuf+VjbnjqDe596UOOnz6Xuu3YqbM8+Non9Hv
```

Comment:
This is causing issues (see the failing notebook execution test). It's the usual thing where the notebook throws an error because the environment `alibi` doesn't exist. It should be changed to `python3` or `ipykernel` (check other notebooks for examples - let's add this to Notion/CONTRIBUTING.md in a follow-up).